addon/components/story-comments.js

```diff
@@ -41,6 +42,10 @@ export default Component.extend({
 
   isShowingComments: computed({
     get() {
+      if (typeof document === 'undefined') {
+        // don't run in fastboot
```

Comment:
So do we lose comments in fastboot? What impact does this have?

Reply:
or are we now pre-fetching them in the comment adapter above

Reply:
we will still have them, this variable just wont be set upon the initial pre-render.